### PR TITLE
Home work

### PR DESCRIPTION
**Pull request by serg1345**

### CODE REVIEW

File: src/by/sergey_l/lesson1/Task3.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+public class Task3 {
+/*
+Имеется 4500 секунд. Вывести в консоль содержащихся в этом количестве секунд:
+А) минут + секунд,
+В) часов + минут + секунд,
+С) дней + часов + минут + секунд,
+D) недель + дней + часов + минут + секунд.
+по аналогии с примером выше.
+*/
+
```
Comment (svetamolchanova) [inline src/by/sergey_l/lesson1/Task3.java]: remove unnecessary spaces (in all places)

File: src/by/sergey_l/lesson1/Task3.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+		
+		string = minutes + " минут " + seconds % 60 + " секунд.";
+		System.out.println(string);
+		string = hours + " часов " + string;
+		System.out.println(string);
+		string = days + " дней " + string;
+		System.out.println(string);
+		string = weeks + " недель " + string;
+		System.out.println(string);
+
```
Comment (svetamolchanova) [inline src/by/sergey_l/lesson1/Task3.java]: unnecessary spaces

File: src/by/sergey_l/lesson1/Task5.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+public class Task5 {
+/*
+Дано любое натуральное 4-х значное число. Верно ли, что все цифры числа
+различны?
+*/
+
+	public static void main(String[] args) {
+		
+//		int number = 1234;
```
Comment (svetamolchanova) [inline src/by/sergey_l/lesson1/Task5.java]: remove unnecessary comment code

File: src/by/sergey_l/lesson3/FromMetodichkaTask12.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    public static void main(String[] args) {
+        int n = 5;
+
+        int sum = 0;
+        int i = 1;
+        do {
+            sum += i * 3;
+            i++;
+        } while (i <= n);
+        System.out.println("Сумма первых " + n + " чисел которые делятся на 3 равна " + sum);
```
Comment (svetamolchanova) [inline src/by/sergey_l/lesson3/FromMetodichkaTask12.java:15]: Incorrect definition of the condition. You need to find the sum of the first three numbers that are divisible by three. Why is the number of values five?

File: src/by/sergey_l/lesson3/FromMetodichkaTask7.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+Имеется прямоугольное отверстие размерами a и b, определить, можно ли его
+полностью закрыть круглой картонкой радиусом r.
+ */
+public class FromMetodichkaTask7 {
+    public static void main(String[] args) {
+        int a = 5;
+        int b = 7;
+        int r = 5;
+        double diagonal = 0;
+
```
Comment (svetamolchanova) [inline src/by/sergey_l/lesson3/FromMetodichkaTask7.java]: Why do we initialize "diagonal" like 0?

File: src/by/sergey_l/lesson3/Task6.java [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+ */
+
+public class Task6 {
+
+    public static void main(String[] args) {
+        for (int i = 2; i <= 100; i += 2) {
+            System.out.println(i);
+        }
+        }
+    }
```
Comment (svetamolchanova) [inline src/by/sergey_l/lesson3/Task6.java:11]: use code formatting, correct indentation

File: src/by/sergey_l/lesson3/FromMetodichkaTask12.java (hunk 4 above)
Comment (serg1345) [reply src/by/sergey_l/lesson3/FromMetodichkaTask12.java:15]: Problem condition: Find the sum of the first n integers that are divisible by 3. I take n = 5

File: src/by/sergey_l/lesson3/FromMetodichkaTask7.java (hunk 5 above)
Comment (serg1345) [reply src/by/sergey_l/lesson3/FromMetodichkaTask7.java]: this is unnecessary, I will fix it

File: src/by/sergey_l/lesson1/Task3.java (hunk 1 above)
Comment (serg1345) [reply src/by/sergey_l/lesson1/Task3.java]: I fixed it

File: src/by/sergey_l/lesson1/Task3.java (hunk 2 above)
Comment (serg1345) [reply src/by/sergey_l/lesson1/Task3.java]: I fixed it

File: src/by/sergey_l/lesson1/Task5.java (hunk 3 above)
Comment (serg1345) [reply src/by/sergey_l/lesson1/Task5.java]: I fixed it

File: src/by/sergey_l/lesson3/Task6.java (hunk 6 above)
Comment (serg1345) [reply src/by/sergey_l/lesson3/Task6.java:11]: I fixed it

File: src/by/sergey_l/lesson3/FromMetodichkaTask12.java (hunk 4 above)
Comment (svetamolchanova) [reply src/by/sergey_l/lesson3/FromMetodichkaTask12.java:15]: ok. I resolve it

**Review by svetamolchanova — APPROVED** [review]

**Pull request merged**